components/pkg-export-helm/src/main.rs

```diff
@@ -190,6 +203,14 @@ fn valid_url(val: String) -> result::Result<(), String> {
     }
 }
 
+fn valid_maintainer(val: String) -> result::Result<(), String> {
+    maintainer::Maintainer::from_str(&val).map(|_| ()).map_err(
```

Comment:
What's the point of the first `map`?

Reply:
there is only 1 map here and that transforms the Ok value to what the `valid_maintainer` needs to return.